src/backend/tests/users.test.ts

```diff
@@ -167,5 +167,13 @@ describe('Users', () => {
       expect(prisma.schedule_Settings.findUnique).toHaveBeenCalledTimes(1);
       expect(res).toStrictEqual(batmanUserScheduleSettings);
     });
+
+    test('setUserScheduleSettings works successfully', async () => {
```

Comment:
Could you add tests for invalid emails/invalid zoom links, and maybe also for non-unique emails?